crates/pathfinder/src/storage/schema/revision_0010.rs

```diff
@@ -299,6 +309,85 @@ mod tests {
                     Ok(())
                 }
             }
+
+            #[derive(Clone, Debug, PartialEq)]
+            pub struct Transaction {
+                pub calldata: Option<Vec<CallParam>>,
+                pub class_hash: Option<ClassHash>,
+                pub constructor_calldata: Option<Vec<ConstructorParam>>,
+                pub contract_address: Option<ContractAddress>,
+                pub contract_address_salt: Option<ContractAddressSalt>,
+                pub entry_point_type: Option<EntryPointType>,
+                pub entry_point_selector: Option<EntryPoint>,
+                pub max_fee: Option<Fee>,
+                pub nonce: Option<TransactionNonce>,
+                pub sender_address: Option<ContractAddress>,
+                pub signature: Option<Vec<TransactionSignatureElem>>,
+                pub transaction_hash: StarknetTransactionHash,
+                pub r#type: Type,
+                pub version: Option<TransactionVersion>,
+            }
+
+            /// Describes L2 transaction types.
+            #[derive(Copy, Clone, Debug, PartialEq)]
+            #[allow(dead_code)]
+            pub enum Type {
+                Deploy,
+                InvokeFunction,
+                Declare,
+            }
+
+            pub struct StarknetEventsTable {}
```

Comment:
I think the same thing is missing in `revision_007.rs` as well then?

At least I recall logging it in #160.

Reply:
I don't quite follow: `revision_0007.rs` already has a copy of `Transaction` & friends that's used for de-serializing transactions from the database.

Reply:
It has:
```rust
1    use crate::storage::state::StarknetEventsTable; <<<
...
...
309                 let serialized_data = StarknetEventsTable::event_data_to_bytes(&event.data);
310                 let serialized_keys = StarknetEventsTable::event_keys_to_base64_strings(&event.keys);
```

Reply:
I've added these missing serializers to this old migration and removed the import of StarknetEventsTable.